docs/README.md

```diff
@@ -21,7 +21,32 @@ more about Community, Certified and Marketplace operators and contribution.
 
 We would love to see your Operator added to this collection. We currently use automated vetting via continuous integration plus manual review to curate a list of high-quality, well-documented Operators. If you are new to Kubernetes Operators start [here](https://sdk.operatorframework.io/build/).
 
-If you have an existing Operator read our contribution guidelines on how to [open a PR](contributing-via-pr.md). Then the community operator pipeline will be triggered to test your Operator and merge a Pull Request.
+If you have an existing Operator read our contribution guidelines on how to [open a PR](users/contributing-via-pr.md). Then the community operator pipeline will be triggered to test your Operator and merge a Pull Request.
+
+## Remove Your Operator
+
+Before you begin, ensure the operator is [FBC-enabled](users/operator-ci-yaml.md#fbc-mode) to support the removal process. You can choose to either remove the entire operator or remove it from specific catalog version(s).
```

Comment:
We have to let the user know that the flag in the ci.yaml is not the only thing that makes the operator FBC. Please also include information that for existing non-FBC operator it needs to be converted (https://redhat-openshift-ecosystem.github.io/operator-pipelines/users/fbc_onboarding/)